lldb/test/API/functionalities/watchpoint/unaligned-spanning-two-dwords/TestUnalignedSpanningDwords.py

```diff
@@ -46,7 +46,9 @@ def test_unaligned_watchpoint(self):
         a_bytebuf_6 = frame.GetValueForVariablePath("a.bytebuf[6]")
         a_bytebuf_6_addr = a_bytebuf_6.GetAddress().GetLoadAddress(target)
         err = lldb.SBError()
-        wp = target.WatchAddress(a_bytebuf_6_addr, 4, False, True, err)
+        wp_opts = lldb.SBWatchpointOptions()
+        wp_opts.SetWatchpointTypeModify(True)
+        wp = target.WatchpointCreateByAddress(a_bytebuf_6_addr, 4, wp_opts, err)
         self.assertTrue(err.Success())
```

Comment:
No need to change this test if it still uses the old API. Revert?

Reply:
I was marking `SBTarget::WatchAddress` as deprecated, so I thought updating the API tests to use the current API was the best choice.  It might be good to keep one using the old API though, to catch a break in it.